docs/topics/Community_Resources.md

```diff
@@ -10,34 +10,35 @@ Many of these libraries are represented in the [unofficial, community-driven Dis
-| [Serenity](https://github.com/serenity-rs/serenity)          | Rust       |
-| [AckCord](https://github.com/Katrix/AckCord)                 | Scala      |
-| [Sword](https://github.com/Azoy/Sword)                       | Swift      |
-| [Discordeno](https://github.com/Skillz4Killz/Discordeno)     | TypeScript |
+| Name                                                         | Language   | Support Server                                |
+| ------------------------------------------------------------ | ---------- | --------------------------------------------- | 
+| [discljord](https://github.com/igjoshua/discljord)           | Clojure    | [Invite](https://discord.gg/284PgKZ)          |
+| [aegis.cpp](https://github.com/zeroxs/aegis.cpp)             | C++        |                                               |
+| [discordcr](https://github.com/discordcr/discordcr)          | Crystal    |                                               |
+| [Discord.Net](https://github.com/RogueException/Discord.Net) | C#         |                                               |
+| [DSharpPlus](https://github.com/DSharpPlus/DSharpPlus)       | C#         | [Invite](https://discord.gg/KeAS3pU)          |
+| [dscord](https://github.com/b1naryth1ef/dscord)              | D          |                                               |
+| [DiscordGo](https://github.com/bwmarrin/discordgo)           | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
+| [DisGord](https://github.com/andersfylling/disgord)          | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
+| [catnip](https://github.com/mewna/catnip)                    | Java       | [Invite](https://discord.gg/yeF2HpP)          |
+| [Discord4J](https://discord4j.com/)                          | Java       | [Invite](https://discord.gg/NxGAeCY)          |
+| [Javacord](https://github.com/Javacord/Javacord)             | Java       | [Invite](https://discord.gg/0qJ2jjyneLEgG7y3) |
+| [JDA](https://github.com/DV8FromTheWorld/JDA)                | Java       | [Invite](https://discord.gg/0hMr4ce0tIk3pSjp) |
+| [discord.js](https://github.com/discordjs/discord.js)        | JavaScript | [Invite](https://discord.gg/bRCvFy9)          |
+| [Eris](https://github.com/abalabahaha/eris)                  | JavaScript |                                               |
+| [Discord.jl](https://github.com/Xh4H/Discord.jl)             | Julia      |                                               |
+| [Discordia](https://github.com/SinisterRectus/Discordia)     | Lua        |                                               |
+| [discordnim](https://github.com/Krognol/discordnim)          | Nim        |                                               |
+| [RestCord](https://www.restcord.com/)                        | PHP        | [Invite](https://discord.gg/0duG4FF1ElFGUFVq) |
+| [discord.py](https://github.com/Rapptz/discord.py)           | Python     | [Invite](https://discord.gg/dpy)              |
```

Comment:
Putting a vanity invite that could potentially expire due to boosting tiers might not be smart. The readme uses this link instead: https://discord.gg/r3sSKJJ